include/materials/norsand.tcc

```diff
@@ -357,8 +357,9 @@ void mpm::NorSand<Tdim>::compute_plastic_tensor(const Vector6d& stress,
       std::sqrt(std::pow(dF_dsigma(0) - dF_dsigma_v, 2) +
                 std::pow(dF_dsigma(1) - dF_dsigma_v, 2) +
                 std::pow(dF_dsigma(2) - dF_dsigma_v, 2) +
-                2 * std::pow(dF_dsigma(3), 2) + 2 * std::pow(dF_dsigma(4), 2) +
-                2 * std::pow(dF_dsigma(5), 2));
+                std::pow(dF_dsigma(3), 2) / 2. +
+                std::pow(dF_dsigma(4), 2) / 2. + std::pow(dF_dsigma(5), 2)) /
+      2.;
```

Comment:
I smell some typo here... Shouldn't it be: 
```suggestion
                std::pow(dF_dsigma(4), 2) / 2. + std::pow(dF_dsigma(5), 2) / 2.);
```
Why the `std::pow(dF_dsigma(5), 2) ` is not divided by 2? And why all the `std::sqrt()` is divided by 2. in the end?

Reply:
Yes fixing... realized that and that is why results are a bit weird